### ISSUE TITLE
standalone / non composer projects

### ISSUE THREAD

**Issue by briceburg** [m2mdas/phpcomplete-extended#44]
do you plan on supporting standalone projects? E.g. scanning vim's working directory, parsing php files? 

this plugin looks great && context aware autocompletion is badly needed; just hoping I can use it on simple projects.

Thanks for your thoughts

**Comment by mkusher** [comment]
@briceburg you can try https://github.com/shawncplus/phpcomplete.vim for this

**Comment by briceburg** [comment]
@mkusher thanks for the heads up. I've tried and wasn't successful at getting context-aware completion going:. E.g.

```
$instance = new SomeClass();
$instance->[autocomplete menu]
```

where [autocomplete menu] has all the possible methods SomeClass provides.

Does it work for you?

**Comment by complex857** [comment]
@briceburg, yes that should be working as long as the plugin can find the source file for `SomeClass`. This usually means that you will need a "tags" file for your project (unless it's defined in the same source file or a built-in). 

These are usually generated with [ctags](http://ctags.sourceforge.net/) (i highly recommend the [patched one](https://github.com/shawncplus/phpcomplete.vim/wiki/Patched-ctags) provided with the plugin for php code), but you can use projects like [php-ctags](https://packagist.org/packages/techlivezheng/phpctags). 

Vim makes [various use](http://vimdoc.sourceforge.net/htmldoc/tagsrch.html#tags) of these "tags" files like: jumping around, searching, previewing the tags under cursor to name a few and generally a useful thing to have around.

If you feel like hacking, you can try to make a converter from a tags file to the index format the phpcomplete-extended plugin expects. I've made the patched one collect docblocks too, so i think this should be doable.
